KDIC/BlogViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  BlogViewController.swift
+//  KDIC
+//
+//  Created by Shaun Mataire on 12/27/15.
+//  Copyright © 2015 Colin Tremblay. All rights reserved.
+//  NOTE: XCode wont let me refector this class is actully for the Blog
+
+import UIKit
+import SafariServices
+
+class BlogViewController: UIViewController,  UITableViewDataSource, UITableViewDelegate {
```

Comment:
Move `UITableViewDataSource` and `UITableViewDelegate` methods to separate class extension. Should look similar to
```
extension BlogViewController : UITableViewDataSource {
  func ... {

  }
}

extension BlogViewController: UITableViewDelegate {
  func ... {

  }
}
```